### DISCUSSION TITLE
Is it possible to add description and documentation in the Swagger ?

### DISCUSSION THREAD

**Discussion by diallonortv** [Azure/data-api-builder#1619] [General]
Hello,
Is it possible to add any description/documentation for columns and entities ? So someone connecting to my api through the Swagger / OpenAPI can understand the meaning of each entity ?

Regards

Vincent

**Comment by seantleonard** [comment]
The only way to do that today would be to download the openapi doc from your dab endpoint and update descriptions yourself, then host that document separately.

**Comment by seantleonard** [comment]
We can explore options to allowing more configurable flexibility with the generated openAPI doc.

**Comment by diallonortv** [comment]
Thanks, i was wondering if it was possible to use the configuration (dab-config.json) to add description for each field and entity, but it's really not a big deal if it's not possible. A good shared word document will do ;-)

Thanks a lot

Vincent
